### DISCUSSION TITLE
How to change the color of the widgets and box window?

### DISCUSSION THREAD

**Discussion by ujur007** [beeware/toga#1698] [Q&A] [answered]
I am new to toga and BeeWare project. I think this is a right place to ask questions if I have.
The issue I am facing is regarding the colors of the widget background and the color of the Buttons.

The following is the code I am using to change the color but unfortunately there is no change from the default gray color. I am using Linux Mint with Beeware application. 

`$briefcase dev`

```
        canvas = toga.Canvas(style=Pack(color='red', background_color = "red"))
        self.main_box = toga.Box()
        self.main_box.style.background_color = "red"
        
        self.btn = toga.Button("Scan", id = colors.RED, on_press = self.Start_scan)
        self.btn.style.padding = 50
        self.btn.style.flex = 1
        self.main_box.add(self.btn)

        self.main_window = toga.MainWindow()
        self.main_window.content = self.main_box
        
        self.main_window.show()
```
My first guess is using the discussion number [#1292](https://github.com/beeware/toga/discussions/1292) is that some os may not support change of colors

**Comment by mhsmith** [comment, marked as answer]
Based on #1203, it looks like the GTK (Linux) backend doesn't currently support `background_color` on `Box` widgets, though it may work on other widgets.

The good news is over the next few months we'll be systematically testing every property of every widget, so issues like this will soon be fixed.

**Comment by ujur007** [comment]
Ok. Thank you for prompt response.
Is there possibility to contribute in testing the software? 
I am developing an app right now with toga and would be good if I can contribute at the same time I develop the app.

**Reply by freakboy3742** [reply]
@ujur007 Yes - we're always looking for contributors. In general, if you hit a feature that isn't implemented, or doesn't look like it has been implemented correctly, we're open to pull requests to address that gap. At present, we don't have a TODO list of things that need to be done; a big part of the testing process we're engaged in at the moment is developing that list of things that need to be addressed.

**Reply by ujur007** [reply]
@freakboy3742 Thanks. Actually I developed a part of my app and it works like a charm but the only downside I could see is GUI is not that impressive which I think can be. Let me know about anything I do...
